docs/for-developers/smart-contract-details.md

```diff
@@ -3,81 +3,51 @@ id: smart-contract-and-subgraph-details
-<tr><td>CONTRACT_BUILDER</td><td>0x058be54b173e324f80ec03672ec14c4d079b82ac</td></tr>
-<tr><td>CONTRACT_TOKEN_TEMPLATE</td><td>0x885e158c7e6605c895e014116661651b22770f7b</td></tr>
-<tr><td>NETWORK</td><td>ethlive</td></tr>
-<tr><td>CYCLE_SECS</td><td>2592000</td></tr>
+<tr><td>Network</td><td>goerli</td></tr>
+<tr><td>Deployer address</td><td>0x341a08926dCa7fa7D135F96E4d76b696e5f6d38d</td></tr>
+<tr><td>Reserve</td><td>0x78EeC20c86e5f40Ceb1b651c38072DF528AE6407</td></tr>
+<tr><td>DripsHub</td><td>0x4FaAB6032dd0264a8e2671F56fd30F69362f31Ad</td></tr>
+<tr><td>DripsHub logic</td><td>0xB79663c5E27C1a2c93aeE2a35b273b0255638267</td></tr>
+<tr><td>DripsHub cycle seconds</td><td>604800</td></tr>
+<tr><td>AddressApp</td><td>0x76F457CD4F60c0a634781bfdB8c5318050633A08</td></tr>
+<tr><td>MetaData</td><td>0xC58cEa5a448A761d2dE80DFa8BfE298780e9dd66</td></tr>
+<tr><td>Commit hash</td><td>682ea8a0c21891622e2a952721ab4fc748306788</td></tr>
 </table>
 
 #### Subgraph Details
-Subgraph: https://thegraph.com/explorer/subgraph/gh0stwheel/drips-on-ethereum
-<br/>Queries (HTTP):     https://api.thegraph.com/subgraphs/name/gh0stwheel/drips-on-ethereum
-<br/>Subscriptions (WS): wss://api.thegraph.com/subgraphs/name/gh0stwheel/drips-on-ethereum
 
+The repository for the subgraph's code can be found 
+<a href="https://github.com/radicle-dev/drips-subgraph/tree/v0.2" target="_blank">here</a>.
 
-## Rinkeby Testnet
+Subgraph page on The Graph's Hosted Mode website:
+<br>https://thegraph.com/hosted-service/subgraph/gh0stwheel/drips-v02-on-goerli
 
-<table>
-<tr><td>CONTRACT_DAI</td><td>0x5592ec0cfb4dbc12d3ab100b257153436a1f0fea</td></tr>
-<tr><td>CONTRACT_DRIPS_HUB</td><td>0xfbcD6918907902c106A99058146CBdBb76a812f6</td></tr>
-<tr><td>CONTRACT_DRIPS_HUB_LOGIC</td><td>0x756E821D9E88D76ef15d2e719bbd4CC3A2167EC1</td></tr>
-<tr><td>CONTRACT_RESERVE</td><td>0x880D5095606c7b541AdDE0F94A6858CbABb63F69</td></tr>
-<tr><td>CONTRACT_RADICLE_REGISTRY</td><td>0xc2a8F699317795956bE5Cc4f9FF61FD4E7667670</td></tr>
-<tr><td>CONTRACT_BUILDER</td><td>0x688662533E0341D518Bcc965525aFc70550CEE39</td></tr>
-<tr><td>NETWORK</td><td>rinkeby</td></tr>
-<tr><td>CYCLE_SECS</td><td>86400</td></tr>
-</table>
+Subgraph endpoints:
+Queries (HTTP):
+<br>https://api.thegraph.com/subgraphs/name/gh0stwheel/drips-v02-on-goerli
 
-#### Subgraph Details
-Subgraph: https://thegraph.com/explorer/subgraph/gh0stwheel/drips-on-rinkeby
-<br/>Queries (HTTP):     https://api.thegraph.com/subgraphs/name/gh0stwheel/drips-on-rinkeby
-<br/>Subscriptions (WS): wss://api.thegraph.com/subgraphs/name/gh0stwheel/drips-on-rinkeby
+Subscriptions (WS):
+<br>wss://api.thegraph.com/subgraphs/name/gh0stwheel/drips-v02-on-goerli
 
+#### How to Get Testnet Eth and DAI
 
-## MATIC (Polygon Mainnet)
+The main faucet for the Goerli testnet is now operated by Alchemy. You will need to sign up for an Alchemy account to get testnet
+ETH and you can do so [here][gf]. Alternately, you can use [this proof-of-work faucet][gp].
```

Comment:
Clicking the links navigates out of the docs page. Do we want this? Maybe worth changing to hrefs with target="_blank"?

Reply:
Unfortunately this is something that Docusauraus doesn't have a good solution for if you're using the [] notation for for linking. Where the links are full, explicit HTML, then it's easy to do, and I've done it in some cases, but it also makes the markdown hard to read and is cumbersome, so I'm somewhat torn. However, I will try to add the target="_blank" where it seems the most impactful.

I definitely find this annoying but don't see a good solution for how to get this for all links to external resources in the docs.